streaming/sender.c

```diff
@@ -879,9 +909,32 @@ void execute_commands(struct sender_state *s) {
                     stream_execute_function_callback(wb, code, tmp);
                 }
             }
-        }
-        else
+        } else if (keyword && strcmp(keyword, PLUGINSD_KEYWORD_REPLAY_CHART) == 0) {
+            const char *chart_id = get_word(words, num_words, 1);
+            const char *start_streaming = get_word(words, num_words, 2);
+            const char *after = get_word(words, num_words, 3);
+            const char *before = get_word(words, num_words, 4);
+
+            if (!chart_id || !start_streaming || !after || !before) {
+                error("STREAM %s [send to %s] %s command is incomplete"
+                      " (chart=%s, start_streaming=%s, after=%s, before=%s)",
+                      rrdhost_hostname(s->host), s->connected_to,
+                      keyword,
+                      chart_id ? chart_id : "(unset)",
+                      start_streaming ? start_streaming : "(unset)",
+                      after ? after : "(unset)",
+                      before ? before : "(unset)");
+            } else {
+                struct replication_request tmp = {
+                        .start_streaming = !strcmp(start_streaming, "true"),
+                        .after = strtoll(after, NULL, 0),
```

Comment:
Please, use `str2ll` instead `strtoll` for these two variables.

Reply:
This is intentional, it supports also parsing numbers in hex. I want to experiment with this to improve performance (hex number parsing is much faster than decimal).